packages/editor/src/client/atoms/descriptionList.tsx

```diff
@@ -44,7 +47,11 @@ export function DescriptionListItem({label, children}: DescriptionListItemProps)
         style={{
           marginLeft: 20
         }}>
-        {children}
+        {message ? (
```

Comment:
Is the warning message only intended to be used to highlight empty property fields? Would there ever be a use case of children being displayed with a warning message (invalid input maybe)? Because then I'd recommend to put both message and the children conditionally without the else statement.

Reply:
It's supposed to get highlighted when information is missing only, at least that's how it's described in the Jira ticket